queries/synths/useBaseFeeRateQuery.ts

```diff
@@ -6,23 +6,36 @@ import { ethers } from 'ethers';
 import { CurrencyKey } from 'constants/currency';
 import { appReadyState } from 'store/app';
 import QUERY_KEYS from 'constants/queryKeys';
+import Wei from '@synthetixio/wei';
 
-const useBaseFeeRateQuery = (currencyKey: CurrencyKey | null) => {
+const useBaseFeeRateQuery = (
+	sourceCurrencyKey: CurrencyKey | null,
+	destinationCurrencyKey: CurrencyKey | null
+) => {
 	const isAppReady = useRecoilValue(appReadyState);
 	const { synthetixjs } = Connector.useContainer();
 
 	return useQuery(
-		QUERY_KEYS.Synths.BaseFeeRate(currencyKey!),
+		QUERY_KEYS.Synths.BaseFeeRate(sourceCurrencyKey!, destinationCurrencyKey!),
 		async () => {
 			const { SystemSettings } = synthetixjs!.contracts;
 
-			const baseFeeRate = await SystemSettings.exchangeFeeRate(
-				ethers.utils.formatBytes32String(currencyKey as string)
-			);
-			return baseFeeRate ? baseFeeRate : null;
+			const [sourceCurrencyFeeRate, destinationCurrencyFeeRate] = (await Promise.all([
+				SystemSettings.exchangeFeeRate(
+					ethers.utils.formatBytes32String(sourceCurrencyKey as string)
+				),
+				SystemSettings.exchangeFeeRate(
+					ethers.utils.formatBytes32String(destinationCurrencyKey as string)
+				),
+			])) as [Wei, Wei];
```

Comment:
Just a nitpick, but I think this might not be `Wei`, but rather `ethers.BigNumber`.

Reply:
Thanks. I will make the change.

Reply:
@koredefashokun I just took another look into the `TradeSummaryCard`, where we passed the `baseFeeRate`. Do you think we should change everything here?
![image](https://user-images.githubusercontent.com/4819006/168157415-744d2159-0bb9-4268-b7f6-7bbda27e2d2f.png)

Reply:
@koredefashokun Change has been made. Thanks.